packages/plans-grid/src/mixins.scss

```diff
@@ -0,0 +1,139 @@
+@import './variables.scss';
+@import 'assets/stylesheets/gutenberg-base-styles';
```

Comment:
I just learned that his fails the build in FSE because it's part of `client`.

My solution for domain-picker was the following

1. Create a `base-styles.scss` file.
2. Put this in it (the contents of `gutenberg-base-styles`).
```scss
@import '~@wordpress/base-styles/z-index';
@import '~@wordpress/base-styles/colors';
@import '~@wordpress/base-styles/variables';
@import '~@wordpress/base-styles/breakpoints';
@import '~@wordpress/base-styles/mixins';
@import '~@wordpress/base-styles/animations';
```
3. Run yarn add `@wordpress/base-styles`

Reply:
It seems like there should be an `index.scss` file in that package so you can just grab all of this. Are we omitting anything or enumerating the environment (vars, mixins…) setup imports?

Reply:
I will create a PR later for `domain-picker` and we can discuss it. Currently I'm just trying to make it run in FSE.

Reply:
Sure 👍 

To clarify, I was referring to the [`@wordpress/base-styles` package](https://github.com/WordPress/gutenberg/tree/master/packages/base-styles). I think the use case is really "import all these files" and it seems like a single import you could reference from your SASS setup to prepare the environment would be preferable.

I'm sure Gutenberg would be amenable to a PR.

Reply:
I can't remember why I didn't add index when creating that package, or if it was just oversight.